WebDevStudios/wd_s
# Theme/general theme enhancements

### DESCRIPTION ###
This adds some refinements/adjustments that have been talked about previously and were easier to roll into one tiny, bulk update than several branches.

1. Adds numeric pagination to archives
2. Removes the taxonomy name from archives, since we're always asked to remove it anyway (So "Category Name" instead of "Category: Category Name" displays on an archive)
3. Replaces instances of `<button>` tags in our blocks/template tags with `<a>` tags with button classes in cases of links that should be styled as buttons. Instances of `<button>` tags which perform an action like a button should remain unchanged.
4. Adds transitions to link and button hovers

### SCREENSHOTS ###
1.
![](https://dl.dropbox.com/s/5yeyw24wnpr11wd/Screenshot%202018-10-08%2013.03.48.jpg?dl=0)

2.
![](https://dl.dropbox.com/s/etsgjetmc2qn76z/Screenshot%202018-10-08%2013.04.10.jpg?dl=0)

3.
![](https://dl.dropbox.com/s/buy8rer9ywzcuxe/Screenshot%202018-10-08%2013.04.40.jpg?dl=0)
![](https://dl.dropbox.com/s/sme2hr82xvj4jbs/Screenshot%202018-10-08%2013.04.49.jpg?dl=0)

4.
![](https://dl.dropbox.com/s/4nqvnvjlpcesm95/wd_s-text-link-hover.gif?dl=0)
![](https://dl.dropbox.com/s/hjp8mdtyg1r9pme/wd_s-button-hover.gif?dl=0)

### OTHER ###

### STEPS TO VERIFY ###
Check out the branch and:
1. View some archive pages or search results
2. View a cat/tag/term archive page
3. View the markup for buttons on the CTA block and anything using the `_s_display_card` template tag (like recent/related posts)
4. Hover links and buttons!

### DOCUMENTATION ###
Will this pull request require updating the wd_s [wiki](https://github.com/WebDevStudios/wd_s/wiki)?

We could probably stand to add the numeric pagination template tag if accepted, just so everyone is a bit more aware of its existence.
